doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -0,0 +1,703 @@
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: ${gatewayName}
+  defaults:
+    rules:
+      authorization:
+        "deny":
+          opa:
+            rego: "allow = false"
+EOF
+```
+
+Let's check our policy was accepted by the controller:
+
+```bash
+kubectl get authpolicy ${gatewayName}-auth -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+```
+
+Setup a `TLSPolicy` for our Gateway:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1alpha1
+kind: TLSPolicy
+metadata:
+  name: ${gatewayName}-tls
+  namespace: ${gatewayNS}
+spec:
+  targetRef:
+    name: ${gatewayName}
+    group: gateway.networking.k8s.io
+    kind: Gateway
+  issuerRef:
+    group: cert-manager.io
+    kind: ClusterIssuer
+    name: ${clusterIssuerName}
+EOF
+```
+
+Let's check our policy was accepted by the controller:
+
+```bash
+kubectl get tlspolicy ${gatewayName}-tls -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+```
+
+Lastly, we'll setup a default `RateLimitPolicy` for our Gateway, with arbitrarily low limits (2 requests in a 10 second window):
+
+```bash
+kubectl apply -f  - <<EOF
+apiVersion: kuadrant.io/v1beta2
+kind: RateLimitPolicy
+metadata:
+  name: ${gatewayName}-rlp
+  namespace: ${gatewayNS}
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: ${gatewayName}
+  defaults:
+    limits:
+      "low-limit":
+        rates:
+        - limit: 2
+          duration: 10
+          unit: second
+EOF
+```
+
+
+To check our rate limits have been accepted, run:
+
+```bash
+kubectl get ratelimitpolicy ${gatewayName}-rlp -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+```
+
+Let's check the programmed state of our gateway listener once more:
+
+```bash
+kubectl get gateway ${gatewayName} -n ${gatewayNS} -o=jsonpath='{.status.listeners[0].conditions[?(@.type=="Programmed")].message}'
+```
+
+We should have a `no errors` response. **Note:** it can take a minute or two for the LetsEncypt ACME certificate to be issued
+
+### Setup our DNS
+
+Having secured and deployed our gateway, the next step involves applying a `DNSPolicy` to direct traffic toward our gateway via the assigned listener hosts. This policy orchestrates traffic flow to the gateways within our clusters. Specifically, it establishes a load-balanced strategy, using a round-robin method for responding to DNS clients. Additionally, we establish a default geo setting. This setting acts as a universal fallback, categorising records under a broad default, ready for future configurations. This setup ensures that should geo-routing be activated on our gateways (to be discussed later), a default will already be in place for any users outside the specified gateway geos, allowing access from any location. We also set a default weight for all records, ensuring uniform application to maintain round-robin distribution.
+
```

Comment:
```suggestion
To check your rate limits have been accepted, enter the following command:

```bash
kubectl get ratelimitpolicy ${gatewayName}-rlp -n ${gatewayNS} -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
```

Check the programmed state of your Gateway listener again:

```bash
kubectl get gateway ${gatewayName} -n ${gatewayNS} -o=jsonpath='{.status.listeners[0].conditions[?(@.type=="Programmed")].message}'
```

You should have a `no errors` response. 

NOTE: It can take a minute or two for the Let's Encypt ACME certificate to be issued.

### Set up your DNS

Having secured and deployed your Gateway, the next step is applying a `DNSPolicy` to direct traffic toward your Gateway via the assigned listener hosts. This policy orchestrates traffic flow to the Gateways in your clusters. Specifically, it establishes a load-balanced strategy, using a round-robin method for responding to DNS clients. 

Additionally, you will establish a default geo setting. This setting acts as a universal fallback, categorizing records under a broad default, ready for future configurations. This ensures that if geo routing is activated on your Gateways, a default is already in place for users outside the specified Gateway geos, allowing access from any location. You will also set a default weight for all records, ensuring uniform application to maintain round-robin distribution.

```